xs/src/libslic3r/SimplePrint.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef slic3r_SimplePrint_hpp_
+#define slic3r_SimplePrint_hpp_
+
+#include "libslic3r.h"
+#include "Point.hpp"
+#include "Print.hpp"
+
+namespace Slic3r {
+
+class SimplePrint {
```

Comment:
I think users of libslic3r would appreciate a doxygen comment explaining what is provided/expected with this class.